src/gigs-board/pages/community/Overview.jsx

```diff
@@ -374,8 +374,18 @@ const Overview = (
   </div>
 );
 
-return widget("components.community.Layout", {
-  label: props.label,
-  tab: "Overview",
-  children: Overview,
-});
+return (
+  <div className="container">
+    <div className="row">
+      <div className="col-xs-12">
+        {widget("components.community.Layout", {
+          label: props.label,
+          tab: "Overview",
+        })}
+      </div>
+    </div>
+    <div className="row">
+      <div className="col-xs-12 col-lg-12">{Overview}</div>
+    </div>
+  </div>
+);
```

Comment:
I don't understand the need to wrap `Layout` around. What is going on here?

Reply:
I can remove that.
originally I added the sidebar to that page as well. forgot to remove the extra wrapper when I removed the sidebar.